data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -11,37 +11,37 @@ Once data is ingested into the app, Members/Users will query for the data to be
 ## Requirements
 
 - Reconciliation is on each record.
-- Reconciled information will apply only to items a User has submitted data for.
-- Anytime a User requests a Reconciliation Report, Logic will be executed for this User on the current dataset stored in the network
+- Reconciled information will apply only to items for which a user has submitted data.
+- When a user requests for Reconciliation Report, logic on the current dataset stored in the network is executed for this user.
```

Comment:
```suggestion
- When a user requests for Reconciliation Report, it will be generated by the current dataset stored in the network for that user.
```